lib/zendesk_apps_tools/server.rb

```diff
@@ -18,7 +18,7 @@ class Server < Sinatra::Base
         end
       end
 
-      ZendeskAppsSupport::Package.new(settings.root).readified_js(nil, settings.app_id, "http://localhost:#{settings.port}/", settings.parameters)
+      ZendeskAppsSupport::Package.new(settings.root).readified_js(nil, settings.app_id, "http://localhost:#{settings.port}/", settings.parameters, params['locale'])
```

Comment:
@princemaple @liulikun how do we feel about doing it this way instead? Corresponding change is [here](https://github.com/zendesk/zendesk_apps_support/pull/83/files#diff-f9dc5da4f07a7cd59a608b6da769f0ecR97)